### DISCUSSION TITLE
[ Enhancement ] - Title Screen Animation & Animated Logo Support

### DISCUSSION THREAD

**Discussion by ChristianSilvermoon** [Wohlstand/TheXTech#365] [Ideas]
What if... When you started up the game... After the load screen

* The title level is revealed with a circle effect (circle zooms out)
* The Logo (`MenuGFX2.png`) drops in from above to it's normal spot while `Sound-17` plays
* It lands in it's normal location with `sounds-37`
* Music fades in

And...

The game supported having an animated graphic such as an Animated PNG for the logo, allowing it to sparkle, or move?

**Comment by 0lhi** [comment]
Moved this to discussions since it's easier to exchange ideas and considerations here.

Speaking for myself, I like the simplicity of the current approach, and how fast it is to start playing. Any change that delays getting into gameplay should be an opt-in imo. Same for the addition of sound effects not associated with player-action.

**Reply by Wohlstand** [reply]
Issues are actual tasks (what to do) and bug reports (which are tasks as they need to be fixed). So, after discussing idea, the issue with a plan of to do, will be made, and assigned with milestone (the next stable release)

**Reply by ChristianSilvermoon** [reply]
> Moved this to discussions since it's easier to exchange ideas and considerations here.
> 
> Speaking for myself, I like the simplicity of the current approach, and how fast it is to start playing. Any change that delays getting into gameplay should be an opt-in imo. Same for the addition of sound effects not associated with player-action.

I would definitely want it to be optional, yeah. I'm mostly just thinking about a bit of artistic presntation, but it should definitely be optional, if even added.

At the very least I think supporting an animated logo graphic would be nice, even without a fancy programatic introduction animation
